descriptors.txt

```diff
@@ -324,10 +333,10 @@ torrid
 trendsetting
 trendy
 triangular
+trolling
```

Comment:
This is very offensive, and I think this should be removed. Should my sister feel bad about herself because she looks like a troll? Please be more sensitive.

Reply:
If I remove `trolling`, will this get merged in?